rbac/providers/azure/aad_auth.py

```diff
@@ -87,3 +90,22 @@ def check_token(self, AUTH_TYPE):
                 "Accept": "application/json",
                 "Host": "graph.microsoft.com",
             }
+
+    def _check_token(self, AUTH_TYPE):
```

Comment:
Hey, I recognize this code  :)

Consider separating the code that issues the GET request from the code that simply switches/returns a known response (maybe pass in graph_token instead and sort out its existence upstream?)

Reply:
Yeah, I'm planning on refactoring this with michael tomorrow when we go over both sync daemons and remove redundancies. Ideally I'll use _check_token() and have it pass the valid token to check_token_POST() and check_token_GET() (which will require a couple small changes of the initial sync). WOuld you rather that be sorted out here?